py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -518,8 +521,13 @@ def compile(
         torch.fx.GraphModule: Compiled FX Module, when run it will execute via TensorRT
     """
 
-    if debug:
-        set_log_level(logger.parent, logging.DEBUG)
+    if kwargs.get("debug", False):
+        warnings.warn(
+            "`debug` is deprecated. Please use `torch_tensorrt.dynamo.Debugger` for debugging functionality",
```

Comment:
consider pointing to the documentation link (or) modify the message "Please use with torch_tensorrt.dynamo.Debugger(...) to wrap your compilation call to enable debugging functionality"